spec/models/membership_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Membership, type: :model do
+  let(:membership) { create(:membership) }
+
+  describe '.mark_as_revoked' do
```

Comment:
```suggestion
  describe '#mark_as_revoked' do
```

As this is an instance method ;)

Reply:
Fixed